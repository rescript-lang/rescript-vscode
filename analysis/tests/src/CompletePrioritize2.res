let ax = 4
let _ = ax
let ax = ""
let _ = ax
module Test = {
  type t = {name: int}
  let add = (ax: t) => ax.name + 1
}
let ax: Test.t = {name: 4}
// ax->
//     ^com

// ax
//   ^com
