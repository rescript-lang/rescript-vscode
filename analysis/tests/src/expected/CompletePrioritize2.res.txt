Complete src/CompletePrioritize2.res 9:7
posCursor:[9:7] posNoWhite:[9:6] Found expr:[9:3->0:-1]
Completable: Cpath Value[ax]->
Package opens Pervasives.JsxModules.place holder
Resolved opens 1 pervasives
ContextPath Value[ax]->
ContextPath Value[ax]
Path ax
CPPipe env:CompletePrioritize2
CPPipe type path:Test.t
CPPipe pathFromEnv:Test found:true
Path Test.
[{
    "label": "Test.add",
    "kind": 12,
    "tags": [],
    "detail": "t => int",
    "documentation": null
  }]

Complete src/CompletePrioritize2.res 12:5
posCursor:[12:5] posNoWhite:[12:4] Found expr:[12:3->12:5]
Pexp_ident ax:[12:3->12:5]
Completable: Cpath Value[ax]
Package opens Pervasives.JsxModules.place holder
Resolved opens 1 pervasives
ContextPath Value[ax]
Path ax
[{
    "label": "ax",
    "kind": 12,
    "tags": [],
    "detail": "Test.t",
    "documentation": null
  }]

